### PR TITLE
COSA-876 Simplify the setup (remove dependencies)

### PR DESCRIPTION
**Pull request by JuanVqz**
### Jira Ticket
[COSA-876](https://ombulabs.atlassian.net/browse/COSA-876)

### Motivation / Context
- COSA-876 Simplify the setup (remove dependencies)
- Remove simplecov gem
- Remove unused gems
- Remove next_rails gem

### Screenshots:

___

I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by arielj** [comment]
We could remove this pre-commit hook too https://github.com/fastruby/fake_points/blob/COSA-876/bin/setup#L36 and the .pre-commit file

Maybe move to SQLite instead of Postgres?

**Comment by JuanVqz** [comment]
> We could remove this pre-commit hook too https://github.com/fastruby/fake_points/blob/COSA-876/bin/setup#L36 and the .pre-commit file
> 
> Maybe move to SQLite instead of Postgres?

Interesting that "SQLite" found the following queries dangerous.

```ruby
# app/models/story.rb
scope :by_position, -> { order("stories.position ASC NULLS FIRST, stories.created_at ASC") }

# app/models/project.rb
  scope :sub_projects_with_ordered_stories, ->(project_id) {
    where(parent_id: project_id)
      .includes(:stories).references(:stories)
      .order("projects.position ASC, stories.position ASC NULLS FIRST")
  }
```

```
ActiveRecord::UnknownAttributeReference:
Dangerous query method (method whose arguments are used as raw SQL) called with non-attribute argument(s): "stories.position ASC NULLS FIRST, stories.created_at ASC".This method should not be called with user-provided values, such as request parameters or model attributes. Known-safe values can be passed by wrapping them in Arel.sql().
```

File: db/schema.rb [hunk 1]
```diff
@@ -54,8 +54,8 @@
     t.integer "sign_in_count", default: 0, null: false
     t.datetime "current_sign_in_at"
     t.datetime "last_sign_in_at"
-    t.inet "current_sign_in_ip"
-    t.inet "last_sign_in_ip"
```
Comment (JuanVqz) [inline db/schema.rb:58]: sqlite3 doesn't support `inet` type, there is a gem but I don't think we want to add an extra gem.

**Review by kaysiz — APPROVED** [review]
LGTM

**Review by arielj — APPROVED** [review]
looks good to me!

**Pull request merged**